featuretools/variable_types/variable.py

```diff
@@ -121,97 +125,196 @@ def interesting_values(self, values):
     _default_pandas_dtype = int
 
 
 class Ordinal(Discrete):
     """Represents variables that take on an ordered discrete value"""
-    _dtype_repr = "ordinal"
+    type_string = "ordinal"
     _default_pandas_dtype = int
 
 
 class Numeric(Variable):
     """Represents variables that contain numeric values
 
+    Args:
+        range (list, optional) : List of start and end. Can use inf and -inf to represent infinity. Unconstrained if not specified.
+        start_inclusive (bool, optional) : Whether or not range includes the start value.
+        end_inclusive (bool, optional) : Whether or not range includes the end value
+
     Attributes:
         max (float)
         min (float)
         std (float)
         mean (float)
     """
-    _dtype_repr = "numeric"
+    type_string = "numeric"
     _default_pandas_dtype = float
 
+    def __init__(self,
+                 id,
+                 entity,
+                 name=None,
+                 range=None,
+                 start_inclusive=True,
+                 end_inclusive=False):
+        self.range = None or []
+        self.start_inclusive = start_inclusive
+        self.end_inclusive = end_inclusive
+        super(Numeric, self).__init__(id, entity, name=name)
+
+    def create_data_description(self):
+        description = super(Numeric, self).create_data_description()
+        description['properties'].update({
+            'range': self.range,
+            'start_inclusive': self.start_inclusive,
+            'end_inclusive': self.end_inclusive,
+        })
+        return description
+
 
 class Index(Variable):
     """Represents variables that uniquely identify an instance of an entity
 
     Attributes:
         count (int)
     """
-    _dtype_repr = "index"
+    type_string = "index"
     _default_pandas_dtype = int
 
 
 class Datetime(Variable):
-    """Represents variables that are points in time"""
-    _dtype_repr = "datetime"
+    """Represents variables that are points in time
+
+    Args:
+        format (str): Python datetime format string documented `here <http://strftime.org/>`_.
+    """
+    type_string = "datetime"
     _default_pandas_dtype = np.datetime64
 
-    def __init__(self, id, entity, format=None, name=None):
+    def __init__(self, id, entity, name=None, format=None):
         self.format = format
-        super(Datetime, self).__init__(id, entity, name)
+        super(Datetime, self).__init__(id, entity, name=name)
 
     def __repr__(self):
-        ret = u"<Variable: {} (dtype: {}, format: {})>".format(self.name, self._dtype_repr, self.format)
+        ret = u"<Variable: {} (dtype: {}, format: {})>".format(self.name, self.type_string, self.format)
 
         # encode for python 2
         if type(ret) != str:
             ret = ret.encode("utf-8")
 
         return ret
 
+    def create_data_description(self):
+        description = super(Datetime, self).create_data_description()
+        description['properties'].update({'format': self.format})
```

Comment:
this is should to go into the `type` field not `properties`

Reply:
Change applied in [this commit.](https://github.com/Featuretools/featuretools/pull/361/commits/bbfb55bf48a0d154860e80a4c53ea92873b4cc28)